app/src/main/java/org/oppia/android/app/survey/surveyitemviewmodel/UserTypeItemsViewModel.kt

```diff
@@ -96,9 +97,14 @@ class UserTypeItemsViewModel @Inject constructor(
   }
 
   private fun getUserTypeOptions(): ObservableArrayList<MultipleChoiceOptionContentViewModel> {
-    val observableList = ObservableArrayList<MultipleChoiceOptionContentViewModel>()
-    observableList += UserTypeAnswer.values()
-      .filter { it.isValid() }
+    val observableList=ObservableArrayList<MultipleChoiceOptionContentViewModel>()
+    val filteredUserTypes=filterByEnumCondition(
+      UserTypeAnswer.values().toList(),
+      {it},
+      {it.isValid()}
+    )
+    observableList+=filteredUserTypes
```

Comment:
Nit: reformat. There needs to be spaces around the +=. Ditto the other files in this PR. Use Ctrl+Alt+L

Reply:
i have made the changes